modules/log6x-loki-sizing.adoc

```diff
@@ -0,0 +1,93 @@
+// Module is included in the following assemblies:
+// * observability/logging/logging-6.0/
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-loki-sizing_{context}"]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `1x.<size>` where the value `1x` is number of instances and `<size>` specifies performance capabilities.
+
+For logging 6.1+, `1x.pico` configuration defines a single Loki deployment with minimal resource and limit requirements, offering high availability (HA) support for all Loki components. This configuration is suited for deployments that do not require a single replication factor or auto-compaction.
+
+Disk requests are similar across size configurations, allowing customers to test different sizes to determine the best fit for their deployment needs.
```

Comment:
Do we want to mention that `1x.pico` is new here again instead of just on the release notes and in the table below?

I think the second paragraph comes out of the short discussion we had in the arch meeting :+1: 

```suggestion
Disk requests are similar across size configurations, allowing customers to test different sizes to determine the best fit for their deployment needs.
```

Reply:
I think the 6.1+ covers that. And you are correct, thanks for explaining!

Reply:
This will be returned to in https://issues.redhat.com/browse/OBSDOCS-1494